src/main/java/com/spotinst/sdkjava/model/ElastigroupStopDeploymentRequest.java

```diff
@@ -0,0 +1,67 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.spotinst.sdkjava.client.rest.JsonMapper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ElastigroupStopDeploymentRequest {
+
+    //region Members
+    @JsonProperty("roll")
```

Comment:
Need to understand more on this?

Reply:
Removed roll property, As it's object in the request.